src/DeviceInterface/AndroidInterface.js

```diff
@@ -109,6 +109,9 @@ class AndroidInterface extends InterfacePrototype {
             }
         })
     }
+
+    /** Noop */
+    firePixel (_pixelParam) {}
```

Comment:
We don't want to fire any js pixel on Android because of how the interfaces are exposed.

Reply:
Previously we were not sending any pixel outside the pulldown UI, so nothing on Android. Since this new pixel is not fired from the UI, but from the detection logic, we need to ensure nothing happens accidentally on Android.